src/app/settings/edit-email/page.module.css

```diff
@@ -0,0 +1,118 @@
+.editaccountconatiner {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: auto;
+  margin-top: 2rem;
+  padding: 1rem;
+}
+
+.editAccounth1 {
+  align-items: center;
+  margin-bottom: 2rem;
+}
+
+.inputfieldContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 2rem auto;
+  width: 100%;
+  max-width: 400px;
+}
+
+.inputfieldContainer input {
+  width: 100%;
+  max-width: 400px;
+  padding: 0.8rem;
+}
+
+.savebttnContainer,
+.changeEmailContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 1.2rem auto;
+  width: 100%;
+  max-width: 400px;
+  margin-bottom: 8rem;
+}
+
+.changeEmailContainer button {
+  width: 100%;
+  height: 100%;
+  max-width: 400px;
+  padding: 0.5rem;
+}
+
+.savebttnContainer button {
+  width: 100%;
+  height: 100%;
+  max-width: 400px;
+  margin-top: 2rem;
+  padding: 0.5rem;
+  background-color: #b0b0b0;
+}
+
+.savebttnContainer.active button {
+  background-color: #b077ff;
+  color: white;
+  cursor: pointer;
+}
+
+.savebttnContainer:hover button {
+  background-color: #b0b0b0 !important; /* this will override the original active button component's css color change on hover */
+}
```

Comment:
![image](https://github.com/user-attachments/assets/18048d5b-6fcb-4454-a13f-6e8b8ac48ca2)


the button should not be gray when I hover over it and there is a change